collector/lib/CollectorConfigInspector.h

```diff
@@ -0,0 +1,30 @@
+#ifndef COLLECTOR_CONFIGINSPECTOR_H
+#define COLLECTOR_CONFIGINSPECTOR_H
```

Comment:
Just for consistency
```suggestion
#ifndef COLLECTOR_CONFIG_INSPECTOR_H
#define COLLECTOR_CONFIG_INSPECTOR_H
```